app.py

```diff
@@ -1,10 +1,11 @@
 from flask import Flask, render_template, request, session, redirect, url_for
 import trello_items as trello
 import view_model as view_model
+from flask_config import Config
 
 def create_app():
     app = Flask(__name__)
-    app.config.from_object('env')
+    app.config.from_object(Config)
```

Comment:
You don't actually need this as the only thing in flask_config is SECRET_KEY which is unused since you don't use encrypted cookies anymore.

So you could delete this and delete the flask_config.py file.